pages/item/_itemId/index.vue

```diff
@@ -54,12 +71,29 @@
             </div>
             <v-col class="mt-2" cols="10">
               <v-row
-                v-if="loaded && item && item.Genres && item.Genres.length > 1"
+                v-if="
+                  loaded &&
+                  item &&
+                  item.GenreItems &&
+                  item.GenreItems.length > 0
+                "
               >
                 <v-col cols="2" class="d-flex align-center pa-0">
                   <label class="text--secondary">Genres</label>
                 </v-col>
-                <v-col cols="7">{{ item.Genres.join(', ') }}</v-col>
+                <v-col cols="7">
+                  <v-chip
+                    v-for="genre in item.GenreItems"
+                    :key="genre"
```

Comment:
```suggestion
                    v-for="(genre, index) in item.GenreItems"
                    :key="index"
```

You treat ``genre`` as an string, but it isn't, so Vue outputs an error in console

Reply:
Used genre.Id in order to account for items updating

Reply:
@MrTimscampi Good point!